shoop/discount_pricing/admin_form_part.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from shoop.admin.form_part import FormPart, TemplatedFormDef
+from shoop.core.models import Shop
+from shoop.discount_pricing.models import DiscountedProductPrice
+
+
+class DiscountPricingForm(forms.Form):
+    def __init__(self, **kwargs):
+        self.product = kwargs.pop("product", None)
```

Comment:
@akx no
@suutari-ai this kind of change would be nice, but would cause me to change how `TemplatedFormDef` initializes its' forms which I'm not too keen into doing.